components/eventing-controller/testing/test_helpers.go

```diff
@@ -269,22 +269,32 @@ func WithNotCleanEventTypeFilter(s *eventingv1alpha1.Subscription) {
 // WithFilter sets the Subscription filter with the given event source and type.
```

Comment:
Comment is not accurate anymore, this function now adds one more filter to the existing ones.